src/libraries/System.IO.FileSystem/tests/FileInfo/AppendText.cs

```diff
@@ -1,28 +1,27 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Text;
 using Xunit;
 
 namespace System.IO.Tests
 {
     public class FileInfo_AppendText : File_ReadWriteAllText
     {
+        protected override bool IsAppend => true;
+
         protected override void Write(string path, string content)
         {
             var writer = new FileInfo(path).AppendText();
             writer.Write(content);
             writer.Dispose();
         }
 
-        [Fact]
-        public override void Overwrite()
+        protected override void Write(string path, string content, Encoding encoding)
         {
-            string path = GetTestFilePath();
-            string lines = new string('c', 200);
-            string appendline = new string('b', 100);
-            Write(path, lines);
-            Write(path, appendline);
-            Assert.Equal(lines + appendline, Read(path));
+            var writer = new StreamWriter(path, IsAppend, encoding);
+            writer.Write(content);
+            writer.Dispose();
         }
```

Comment:
Why this type overrides it to use StreamWriter? That contradicts this comment in the base type:
https://github.com/dotnet/runtime/blob/e0dafc540a0d6c532fa3f97f269a1b59ecd6e8ca/src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllText.cs#L174